docs/source/guides/tools/BIKVideoModding.rst

```diff
@@ -0,0 +1,37 @@
+BIK Video Modding
+===============================================
+
+Prequisites to Video Modding
+~~~~~~~~~~~~~
+
+- `The RAD Video Tools <http://www.radgametools.com/bnkdown.htm>`__ installed on your PC
+.. note::
+    The ZIP file containing the installer is password protected, you can find it on the download page
```

Comment:
make ``download page`` a hyper link too, it might not be clear its the same thing ( if you are not the brightest )